transmitter.js

```diff
@@ -1,31 +1,19 @@
 let transmissionFailureCount = 0;
-
-function networkTransmitStub(celcius) {
-    console.log(`Temperature to transmit: ${celcius} celcius`);
-
-    // The intention here is to transmit the temperature over a network.
-    // However, this is a stub, so we just print the message above
-    // and give back a return code.
-
-    // Return 200 for ok
-    // Return 500 for not-ok
-    // This stub always succeeds and returns 200
-    return 200;
+function networkTransmitStub(celsius) {

```

Comment:
`transmitInCelsius` is production code, but this file includes the stub as well. This means customers who want `transmitInCelsius` will need to take the stub as well, though they don't want it. Move the stub to the test file.
Then you will need to manage without requiring the stub in `transmitter.js` directly. To solve this, lookup "dependency injection"